deploy/backplane/srep/hypershift/management-cluster/10-srep-management-cluster-project.ClusteRole.yml

```diff
@@ -0,0 +1,97 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-srep-management-cluster-project
+rules:
+- apiGroups:
+  - work.open-cluster-management.io
+  resources:
+  - appliedmanifestworks
+  - operator
+  - work
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - cluster.open-cluster-management.io
+  resources:
+  - clusterclaims
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - operator.open-cluster-management.io
+  resources:
+  - klusterlets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - policy.open-cluster-management.io
+  resources:
+  - policies
+  - configurationpolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - hypershift.openshift.io
+  resources:
+  - awsendpointservices
+  - hostedclusters
+  - hostedcontrolplanes
+  - nodepools
+  verbs:
+  - patch
```

Comment:
SRE's ability to patch these resources is a concern to me.  They are managed from the SC via ManifestWorks.  Any changes made by SRE will be reconciled.  

```suggestion
```